courses/machine_learning/deepdive2/time_series_prediction/4_modeling_keras.ipynb

```diff
@@ -0,0 +1,1261 @@
+{
```

Comment:
<p>instead of shutil, just do ! rm -r $LOGDIR</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/689/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='72'/>

Reply:
<p>decided to get rid of this section since it's less relevant with the new dataset and they already have examples of similar input functions. </p>